typescript/infra/config/environments/mainnet3/chains.ts

```diff
@@ -41,10 +41,10 @@ export const ethereumMainnetConfigs: ChainMap<ChainMetadata> = {
       ...chainMetadata.ethereum.blocks,
       confirmations: 3,
     },
-    transactionOverrides: {
-      maxFeePerGas: 150 * 10 ** 9, // gwei
-      maxPriorityFeePerGas: 5 * 10 ** 9, // gwei
-    },
+    // transactionOverrides: {
```

Comment:
Can you confirm you want this change checked in? I agree conceptually a hard-coded eth gas fee is brittle

Reply:
I needed this to get the deploy to work